### ISSUE TITLE
Do you evaluate the model with the checkpoint by epoch-60 ?

### ISSUE THREAD

**Issue by YunpengZhai** [huanghoujing/beyond-part-models#14]
Emmmm, I found the rank1 and mAP drop a little, maybe because overfitting.

**Comment by huanghoujing** [comment]
Yes, I test the model on test set ONLY when the training process ends. Maybe intermediate results are better, I am not sure :-)

**Comment by Gavin666Github** [comment]
![pcb](https://user-images.githubusercontent.com/35061171/41825864-4ad82152-7857-11e8-8f42-f1520741f0ac.png)

the results of training and testing are as above ,but set test only ,it turned out pretty bad.see figure below:

![pcb_test](https://user-images.githubusercontent.com/35061171/41825865-5042ea0a-7857-11e8-8853-fc2d52bd38cb.png)


is there anyone who knows what happened?

**Comment by huanghoujing** [comment]
In your log, there is warning about loading model weight. You have to make this correct first.

**Comment by huanghoujing** [comment]
@Gavin666Github It seems that your problem is related to the difference between `model_weight_file` and `ckpt_file` in my code. I think [this issue](https://github.com/huanghoujing/beyond-part-models/issues/2) should solve your problem.

**Comment by Gavin666Github** [comment]
@huanghoujing Thank you for the guidance. I'll try to do it  with new commands.

**Comment by Gavin666Github** [comment]
![pcb_test1](https://user-images.githubusercontent.com/35061171/41897492-15c12e9c-795a-11e8-95ac-daa7c8ef0f03.png)


@huanghoujing  As shown in the picture, the problem has been solved perfectly, and it is very powerful. Thank you very much indeed!
